arduino-ide-extension/src/node/i18n/localization-backend-contribution.ts

```diff
@@ -0,0 +1,39 @@
+import * as express from 'express';
+import { inject, injectable } from '@theia/core/shared/inversify';
+import { LocalizationBackendContribution as TheiaLocalizationBackendContribution } from '@theia/core/lib/node/i18n/localization-backend-contribution';
+import { PluginDeployer } from '@theia/plugin-ext/lib/common/plugin-protocol';
+import { PluginDeployerImpl } from '@theia/plugin-ext/lib/main/node/plugin-deployer-impl';
+import { Deferred } from '@theia/core/lib/common/promise-util';
+
+@injectable()
+export class LocalizationBackendContribution extends TheiaLocalizationBackendContribution {
+  @inject(PluginDeployer)
+  protected readonly pluginDeployer: PluginDeployerImpl;
+
+  private initialized = new Deferred<void>();
+
+  override async initialize(): Promise<void> {
+    this.pluginDeployer.onDidDeploy(() => {
```

Comment:
It's an OK solution for now, but the deferred promise will never resolve if the plugins are deployed by the time this `initialize` function is called.

Reply:
@kittaakos is that a likely scenario? (getting to this initialize post deploy)

Reply:
No, I do not think so.

Based on the method names (`initialize` and `start`), I do not think it will cause an issue at runtime, but I did not thoroughly check the Theia code.

Reply:
@AlbyIanna lets keep note of this, maybe we'll get a hint [here](https://github.com/eclipse-theia/theia/pull/11472) anyway.